Fix11 (#3403)
modelica/ModelicaSpecification: chapters/connectors.tex

\begin{lstlisting}[language=modelica]
connect(a, b);
\end{lstlisting}

> one connection set is generated for every pair of corresponding primitive components of \lstinline!a! and \lstinline!b! together with an indication of whether they are from an inside or an outside connector.

Reviewer: Tone down emphasis on the count:
```suggestion
a connection set is generated for every pair of corresponding primitive components of \lstinline!a! and \lstinline!b! together with an indication of whether they are from an inside or an outside connector.
```

Author: I'm not sure which variant is best; I can understand that being clear that it is one for each pair might avoid ambiguities.

Reviewer: Maybe _a… each_ is better?
```suggestion
a connection set is generated for each pair of corresponding primitive components of \lstinline!a! and \lstinline!b! together with an indication of whether they are from an inside or an outside connector.
```

> each of their primitive components will generate one connection set comprised of the corresponding primitive components of \lstinline!a! and \lstinline!b! together with an indication of whether they are from an inside or an outside connector.

Reviewer 2: This is repetitive: primitive components will generate one ... set ...comprised of ... primitive components. Perhaps:
```suggestion
one connection set is generated for every pair of corresponding primitive components of \lstinline!a! and \lstinline!b! together with an indication of whether they are from an inside or an outside connector.
```

Author: Ah, yes, that's better.
